UI/src/components/molecules/RipaStopDate.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="ripa-stop-date tw-p-4">
+    <ripa-form-header
+      title="Date of Stop"
+      required
+      subtitle="§999.226(a)(10)">
+    </ripa-form-header>
+
+    <v-checkbox
+      v-model="checkbox"
+      dense
+      cbLabel: "Stop in response to Call for Service"
+      cbValue: "Stop in response to Call for Service"
+      hide-details>
+    </v-checkbox>
+
+    <ripa-date-picker
+      v-model="datePicker"
+      label="Date of Stop">
+    </ripa-date-picker>
+
+    <ripa-time-picker
+      v-model="timePicker"
+      label="Time of Stop">
+    </ripa-time-picker>
+
+    <ripa-number-input
+      v-model="duration"
+      label= "Stop Duration"
+      required>
+    </ripa-number-input>
+  </div>
+</template>
+
+<script>
+import RipaFormHeader from '@/components/molecules/RipaFormHeader'
+import RipaCheckbox from '@/components/atoms/RipaCheckbox'
+import RipaDatePicker from '@/components/atoms/RipaDatePicker'
+import RipaTimePicker from '@/components/atoms/RipaTimePicker'
+
+export default {
+  name: 'ripa-stop-date',
+
+  components: {
+    RipaFormHeader,
+    RipaCheckbox,
+    RipaDatePicker,
+    RipaTimePicker,
+  },
+
+  data() {
+    return {
+      valid: true,
+      duration: '',
+      checkbox: false,
+      datePicker: format(new Date(), 'yyyy-MM-dd'),
+      timePicker: format(new Date(), 'h:mm'),
+    }
+  },
+
+  computed: {
+    getModel() {
+      return {
+        duration: this.duration,
+        checkbox: this.checkbox,
+        explanation: this.explanation,
+        datePicker: this.datePicker,
+        timePicker: this.timePicker
+      }
+    },
+  },
```

Comment:
I would use the model/viewModel pattern similar to what you see in the atom form components.

You can reference model.duration, for example.

The computed model using get and set, the set would be used to emit the input. Let me come up with an example tonight on this pattern since we don't have one currently.

Reply:
an example would be helpful! thank you

Reply:
Hey @stephsmith0101 . Take a look at the updated RipaStopReason component in dev. I am using @input="handleInput" for each form component. When a form component is updated, it will call the handleInput method, format the object, and emit it so the parent can use it. If you look at the RipaStopReason.stories.js file, you can see that I am default the explanation, you see the object markup below the RipaStopReason component as part fo the {{ stopReason }}.